marketplace-service/src/main/java/com/axonivy/market/github/service/impl/GHAxonIvyProductRepoServiceImpl.java

```diff
@@ -213,10 +221,31 @@ private void getDependencyContentsFromProductJson(ProductModuleContent productMo
         productModuleContent.setArtifactId(artifact.getArtifactId());
         productModuleContent.setType(artifact.getType());
         productModuleContent.setName(artifact.getName());
+
```

Comment:
Too complicated method with 4 "if" layers. Please split it to smaller ones that we can easy to test all of branch and maintain